### ISSUE TITLE
Project/Manifest

### ISSUE THREAD

**Issue by felipenoris** [queryverse/LibXLS.jl#18]
Can we add Project/Manifest for this now? I find it easier to activate a separate environment.

Given that packages with `REQUIRE` will be no longer accepted, I think this move makes sense.

**Comment by davidanthoff** [comment]
Yes, maybe we should merge it on Mon, when stuff is switched over? I think Stefan wrote that he might be writing a script that auto-converts, so that might also make it easier? If there is going to be a script, I'm going to run it on all packages in this org, and then it would probably be easiest if we do it all automated.
